monitoring/logging.sh

```diff
@@ -236,6 +239,13 @@ RMQMessageRunProp()
      --Transmit "${RMQMessaging_Transmit}"         \
      --input_filename "$RPDIR/run.properties"      \
      --output_filename "$RPDIR/run.properties.json" >> ${SYSLOG} 2>&1
+
```

Comment:
If you capture `$?` with a variable, then you can use it later. E.g.,
```
  esta=$?
  if [ $esta == 0 ] ; then
    allMessage "${RMQMessaging_Script_RP} returned a 0 status."
  else
    allMessage "${RMQMessaging_Script_RP} returned an error status ($esta), with these parameters: $ASGS_PID, $INSTANCENAME, $RPDIR, ${RMQMessaging_Transmit}" 
  fi 
```

Reply:
OK